modules/compliance-tests/src/smithy4s/compliancetests/internals/Assertions.scala

```diff
@@ -50,13 +50,26 @@ private[internals] object assert {
     }
   }
 
-  def eql[A](expected: A, actual: A): ComplianceResult = {
-    if (expected == actual) {
-      success
-    } else {
+  def arrEql(expected: Array[Any], actual: Array[Any]): ComplianceResult = {
+    if (java.util.Arrays.deepEquals(expected, actual)) success
+    else
       fail(
-        s"Actual value: ${pprint.apply(actual)} was not equal to ${pprint.apply(expected)}."
+        s"Arrays are not equal  ${pprint.apply(actual)} \n expected:  ${pprint.apply(expected)}"
       )
+  }
+  Tuple2
```

Comment:
**issue:** I don't think you meant this to end up here :D

Reply:
no there is a bunch of garbage here , was just opened to share code with @Baccata